src/utils/update/trust.model.fetch.ts

```diff
@@ -79,6 +79,14 @@ export const mapTrustData = (trustData: TrustData, appData: ApplicationData) =>
     stillInvolved = "";
   }
 
+  let unableToObtainAllInfo = "";
```

Comment:
It would be preferable to leave a comment saying that this part of the code must be removed when the SDK has been updated.

Reply:
added comment